src/app/components/card/Card.jsx

```diff
@@ -14,9 +14,7 @@ const Card = () => {
           <span className={styles.category}>CULTURE</span>
         </div>
         <Link href="/" className={styles}>
-          <h1>
-            Neque porro quisquam est qui dolorem ipsum quia dolor sit amet.
-          </h1>
+          <h1>{item.title}</h1>
```

Comment:
the title is not being called, tried to create a mockdata on prisma but have some bugs there.